src/utils/chart.js

```diff
@@ -97,13 +97,19 @@ export function LineChartBuilder(title,data,label){
         width:'100%'
       },
       stroke: {
-        curve: 'straight'
+        curve: 'straight',
+        width: 2
       },
       dataLabels: {
         enabled: false
       },
       xaxis: {
-        categories: ''
+        type: 'category',
+        overwriteCategories: ['1', (data.length)/4, (data.length)/2, (data.length*3)/4, data.length],
```

Comment:
Why call it overwriteCategories when there was already an item called categories before?
By hard coding in it at 1/4, 1/2, 3/4 it doesnt make it flexible if for future use, it is easier to comprehend specific intervals like every 50 or 20. Existing screen shot has it at roughly each 80 which isn't all that common when looking at data.

Reply:
How's this?
![image](https://user-images.githubusercontent.com/96130742/236688309-17261474-b689-4940-809a-e8b53967b9dc.png)
![image](https://user-images.githubusercontent.com/96130742/236688413-991b0541-c620-4b6c-b7ee-1ff4e484527a.png)

Reply:
and the specific interval is passed through as a parameter

Reply:
That's fine. Just make sure to keep the one value at the beginning. Remove the comments.

Reply:
Just couldn't get anything working with `categories` for some reason, `overwriteCategories` worked and I have no idea why!